src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -51,12 +57,13 @@ public function __construct($redis, array $options = [])
             throw new \InvalidArgumentException(sprintf('%s() expects parameter 1 to be Redis, RedisArray, RedisCluster or Predis\ClientInterface, %s given', __METHOD__, \is_object($redis) ? \get_class($redis) : \gettype($redis)));
         }
 
-        if ($diff = array_diff(array_keys($options), ['prefix'])) {
+        if ($diff = array_diff(array_keys($options), ['prefix', 'expiretime'])) {
```

Comment:
I'd name it `lifetime` (or `maxlifetime` - or `ttl`) - `expiretime` is new vocabulary, it can be surprising thus not easier to understand.